contributingGuides/CONTRIBUTING.md

```diff
@@ -170,3 +170,10 @@ Additionally if you want to discuss an idea with the open source community witho
 #### Important note about JavaScript Style
 - Read our official [JavaScript and React style guide](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md). Please refer to our Style Guide before asking for a review.
 - We have nothing against Prettier or any other automatic style fixers, but we generally don't use them here at Expensify. Do not use Prettier. The style changes these tools enforce don't always align with the ones we recommend and require in our eslint configs and can result in unnecessary changes for our reviewers. Ignoring this advice will ultimately make your changes take longer to review as we will ask you to undo any style changes that are not related to the important changes you are making.
+
+#### For external agencies that Expensify partners with
+Follow all the above above steps and processes. When you find a job you'd like to work on:
+- Post “I’m from [agency], I’d like to work on this job”
+  - If no proposals have been submitted by other contributors, BugZero (BZ) team member or an internal engineer will assign the issue to you. 
+  - If there are existing proposals, BZ will put the issue on hold. Contributor+ (C+) will review the existing proposals.  If a contributor’s proposal is accepted then contributor will be assigned to the issue.  If not the issue will be assigned to the agency-employee. 
+- Once assigned follow the steps [here](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#propose-a-solution-for-the-job) to submit your proposal 
```

Comment:
Hey @mallenexpensify just want to confirm if this is correct since this is different from the [slack](https://expensify.slack.com/archives/C01SKUP7QR0/p1679430232902819) version.

Shouldn't we be mentioning the P.R. creation in this step (like the following message)? Or will the agency members have to create a Proposal after being assigned to the issue?
"Once assigned, submit your P.R. following [the steps](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#begin-coding-your-solution-in-a-pull-request) in CONTRIBUTING.md and a C+ will review it shortly"

Reply:
Bump @mallenexpensify on this : )

Reply:
Agency-member should submit a proposal before the PR.  I originally thought they just submitted a PR, without a proposal, but I was incorrect. Let me know if there's somewhere I need to update this.

Reply:
Makes total sense but I wanted to confirm, thks! 
No need for changes, merging this here